src/components/mobile-icons/MobileIcon.jsx

```diff
@@ -0,0 +1,77 @@
+// @flow strict
+
+import React from 'react';
+import classNames from 'classnames';
+import type {IconColorType} from '../icons/Icon';
+
+export type IconTypeType =
+  | 'answer-bubble'
+  | 'ask-bubble'
+  | 'clear'
+  | 'comment'
+  | 'crown'
+  | 'crown-outline'
+  | 'cup'
+  | 'flashlight-off'
+  | 'flashlight-on'
+  | 'heart-outline'
+  | 'keyboard'
+  | 'options'
+  | 'sent'
+  | 'share-on-ios'
+  | 'star-outline';
+
+export type SizeType = 'small' | 'medium' | 'normal';
+
+export const TYPE: {[name: string]: IconTypeType, ...} = {
+  ANSWER_BUBBLE: 'answer-bubble',
+  ASK_BUBBLE: 'ask-bubble',
+  CLEAR: 'clear',
+  COMMENT: 'comment',
+  CROWN: 'crown',
+  CROWN_OUTLINE: 'crown-outline',
+  CUP: 'cup',
+  FLASHLIGHT_OFF: 'flashlight-off',
+  FLASHLIGHT_ON: 'flashlight-on',
+  HEART_OUTLINE: 'heart-outline',
+  KEYBOARD: 'keyboard',
+  OPTIONS: 'options',
+  SENT: 'sent',
+  SHARE_ON_IOS: 'share-on-ios',
+  STAR_OUTLINE: 'star-outline',
+};
+
+type PropsType = {
+  type: IconTypeType,
+  size?: SizeType,
+  color?: IconColorType,
+  className?: string,
+  ...
+};
+
+const MobileIcon = ({
+  type,
+  size = 'normal',
+  color = 'light',
```

Comment:
It's not a big deal but I'm slightly confused why sometimes color is "white" and sometimes color is "light", light/dark seems to be a kind of "mode", not a color 🤔

Reply:
yeah, it's confusing! but here I'm using the same color values as other icon sets so it's a global issue that we can discuss with design team.

Reply:
ok!